src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -292,7 +292,7 @@ async Task<DurableHttpResponse> IDurableOrchestrationContext.CallHttpAsync(Durab
                 }
 
                 this.IncrementActionsOrThrowException();
-                await this.InnerContext.CreateTimer(fireAt, CancellationToken.None);
+                await this.InnerContext.CreateTimer(fireAt: fireAt, state: true, cancelToken: CancellationToken.None);

```

Comment:
Since we don’t use the result here, I assume any value passed to state is fine. But let me know if you’d prefer a different value.